package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "",
   "dependencies": {
-    "@aragon/os": "3.1.9",
+    "@aragon/os": "^4.0.0-alpha.2",
     "@aragon/client": "^1.0.0-beta.8",
     "@aragon/ui": "^0.16.0",
     "react": "^16.3.2",
```

Comment:
I would still add truffle here, since everyone's more familiar with using it directly. I'm not sure how well battle tested our wrapping around truffle is (I remember having issues with tests, see https://github.com/aragon/aragon-cli/issues/97).